src/kirby/components/badge/badge.component.tns.html

```diff
@@ -0,0 +1 @@
+<Label [text]="text" [colortype]="colortype"></Label>
```

Comment:
```suggestion
 <Label *ngIf="text" [text]="text" [colortype]="colortype"></Label>
 <ng-content *ngIf="!text"></ng-content>
```

Reply:
Needed to add the AbsoluteLayout container again to accommodate this request.